.github/workflows/e2e.yaml

```diff
@@ -1,18 +1,26 @@
+---
+name: End to End Tests
+
 on:
   pull_request:
 
-name: End to End Tests
 jobs:
   e2e:
+    name: E2E
     timeout-minutes: 30
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         globalnet: ['', 'globalnet']
     steps:
-      - name: Checkout the repository
-        uses: actions/checkout@master
+      - name: Check out the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Fetch git tags
+        run: git fetch origin +refs/tags/*:refs/tags/*
```

Comment:
Why is this actually necessary?

Reply:
It might not be, I will check. We do the same in the Operator, so if it's not necessary I guess we should remove it there as well.

Reply:
See also https://github.com/submariner-io/submariner-operator/pull/762.

Reply:
On the operator it's actually necessary since otherwise the CI can't calculate the version, but anywhere else it's not necessary to clone with depth 0 or to fetch the tags, hence my comment.

Perhaps in the operator it's better to comment about this in the GHA file so it's clearer why it's done.

Reply:
@mkolesnik I think it's necessary in some places in the Operator, but not in the locations removed in https://github.com/submariner-io/submariner-operator/pull/762. Do you think those are necessary too?